layouts/about/single.html

```diff
@@ -125,7 +125,33 @@ <h1 class="text-white about-pg-head">Want to submit a talk/workshop for the conf
                           class="yellow-text yellow-hovertext"><u>CFP page </u></a> and <a href="https://in.pycon.org/blog/2021/cfp-announcement.html"
                           class="yellow-text yellow-hovertext"><u> blog</u> </a>for more details. </p>
 
+                  
+                
+                <h1 class="text-white about-pg-head">What are BoFs and Poster Sessions? How one can submit these Proposals? </h1>
 
+                  <p class="text-grey pt-3">
+
+                      Birds of a Feather sessions (or BoFs) are informal gatherings of like-minded individuals 
+                    
+                      who wish to discuss a certain topic without a pre-planned agenda. BoFs are informal or 
+                    
+                      relaxed pannel discussions with active participation from the audiance. </p>
+                
+                
+                  <p class="text-grey pt-3">
+
+                      Posters are graphical summary of projects or ideas. The poster session provides an opportunity to 
+                    
+                      network with the conference attendees and acts as a starting point for further discussions. This being an online event, 
+                    
+                      poster presentation will be done with a pre-recorded video of 5-10 minutes which will be followed by a short Q & A session.</p>
+                    
+                  <p class="text-grey pt-3">
+
+                      To submit the proposal visit <a href="https://in.pycon.org/cfp/bofs-and-posters-2021/proposals/"
+                          class="yellow-text yellow-hovertext"><u>BoF and Poster Submission Page</u></a> and read our <a href=" "
```

Comment:
Either we need to wait till the blog is merged and published and add the blog link here, or we can just remove the reference to blog in this sentence. @wavicles what do you think?

Reply:
I guess the blog is ready. Let's add the link and push.